packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -371,6 +378,18 @@ export class ProjectModel {
         userUuid: string,
         organizationUuid: string,
         data: CreateProject,
+    ): Promise<string> {
+        return this.createWithOptionalCredentials(
+            userUuid,
+            organizationUuid,
+            data,
+        );
+    }
+
+    async createWithOptionalCredentials(
```

Comment:
I created a new model for creating a project without credentials for our use case. Everywhere else still uses the required credentials model